### PR TITLE
Feature/issue 15

### PR DESCRIPTION
**Pull request by ovaistariq**

### CODE REVIEW

File: omnibus/package-scripts/twindb-backup/postinst [hunk 1]
```diff
@@ -22,11 +22,11 @@ mkdir -p ${LOG_DIR} || error_exit "Cannot create ${LOG_DIR}!"
 
 # setup config
 mkdir -p ${CONFIG_DIR} || error_exit "Cannot create ${CONFIG_DIR}!"
-cp $INSTALL_DIR/support/twindb-backup.cfg $CONFIG_DIR/
+mv $INSTALL_DIR/twindb-backup.cfg $CONFIG_DIR/twindb-backup.cfg
```
Comment (akuzminsky) [inline omnibus/package-scripts/twindb-backup/postinst]: Will new package version overwrite the config file?

Comment (ovaistariq) [inline omnibus/package-scripts/twindb-backup/postinst]: Very good question, let's test?

Comment (ovaistariq) [inline omnibus/package-scripts/twindb-backup/postinst]:
```
[root@617be3fbb772 /]# grep backup_mysql /etc/twindb/twindb-backup.cfg
backup_mysql=yes
[root@617be3fbb772 /]# yum install pkg/twindb-backup-2.8.1-1.x86_64.rpm
Complete!
[root@617be3fbb772 /]# grep backup_mysql /etc/twindb/twindb-backup.cfg
backup_mysql=no
```

so it does overwrite the config on upgrade

**Comment by ovaistariq** [comment]
@akuzminsky what do you think now?

**Pull request merged**